### ISSUE TITLE
Is this still active

### ISSUE THREAD

**Issue by joshua1** [stagas/drama#5]
Hi @stagas ,
I have been looking a lot at the actor model for programming (erlang,scala akka, akka.net) and came across this while searching for a js implementation. i wish to know if this is still active.
Thanks

**Comment by stagas** [comment]
no but feel free to use the source or do whatever with it :)

**Comment by pligor** [comment]
Just do it with [Akka](http://akka.io/) instead. Use [Scalatra](http://www.scalatra.org/) as a frontend for Akka and have your node.js communicate with Akka through that. No need to reinvent the wheel. This way you get to keep your nodejs web server and any capabilities that comes with. Always in my humble opinion and please note that I haven't tried it yet myself but seems like a nice way to go

**Comment by stagas** [comment]
When you never reinvent the wheel, you never really learn how the wheel works, which was the reason for making this. I reimplement things as a learning exercise. Also, you have the chance of inventing a better wheel, and controlling that wheel at the lowest level instead of working around. Just my opinion, Akka is great, I'm sure Scalatra and others are too. But "don't reinvent the wheel" is bad advice.

**Comment by pligor** [comment]
You are correct on that part. My answer is still valid for the use case where you care for production and not having many resources in time and money for research and development. You should always explore new things when you can, I am never against that :) But a team of people is required to provide something stable and maintain it in the long run. Thanks for answering back
